guidelines.md

```diff
@@ -0,0 +1,66 @@
+# Guidelines
+---
+*All references are given for each chapter.*
+
+## Programming Style
+
+[This link](https://google.github.io/styleguide/javaguide.html) provides all information needed.
+
+## Javadoc
+```java
```

Comment:
If this code block is copied from some documentation, you can include a link to it and get rid of this section entirely. Less is more

Reply:
I modified some part of the code blocks so they are not the same as those from the source.